### DISCUSSION TITLE
newbee q: flutterFlow with appWrite - how to?

### DISCUSSION THREAD

**Discussion by PiotrEsse** [appwrite/appwrite#3607] [Q&A] [answered]
Hi, please point me to some resources/guides as how to use appwrite with flutterflow?
I am a begginer in nocode (but have some knowledge about coding) and I am trying to build an mobile app and as a backend use just API. I am considering to use just API calls and don't mess with SDK installation either on AW nor FF. 

I ve installed appWrite selfhosted but really cant find any usefull information how to proceed.

**Comment by stnguyen90** [comment, marked as answer]
Our docs (e.g. [Databases API](https://appwrite.io/docs/client/databases)) gives all the details about the URLs, request methods, and parameters. 

Here's an additional article on making API calls without an SDK: https://medium.com/appwrite-io/how-to-connect-with-the-appwrite-backend-server-directly-using-only-the-appwrite-http-api-1e720b567395. 

Finally, our API specs can be found [here](https://github.com/appwrite/appwrite/tree/master/app/config/specs).

**Reply by stnguyen90** [reply]
For detailed guidance on connecting FlutterFlow to Appwrite, check out [this](https://medium.com/@stnguyen90/connecting-flutterflow-to-appwrite-56abdbcbbd92) article.

**Comment by PiotrEsse** [comment]
Ive got much to learn. Thank You!
